### PR TITLE
Update Revolution's app store page from Play Store to F-Droid

### PR DESCRIPTION
**Pull request by Nuc1eoN**
Since the Revolution IRC client has an official F-Droid store release we should prefer linking to that, instead of the corporate Google Play store (also the Play store does't even care to show the projects license).

Alternatively we could also link both, Play Store and F-Droid. Or even just link to Revolutions official web presence.

But personally I'd prefer just exchanging the link for the F-Droid page as done in my PR :)

### CODE REVIEW

File: content/_guides/clients.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ Some commonly chosen graphical clients for personal computers are are
 [Textual](https://www.codeux.com/textual/) (MacOS).
 
 On phones,
-[Revolution](https://play.google.com/store/apps/details?id=io.mrarm.irc) is
+[Revolution](https://f-droid.org/de/packages/io.mrarm.irc/) is
```
Comment (emersion) [inline content/_guides/clients.md:43]: `/de/` makes the F-Droid website appear in German. s/de/en/?

Comment (Nuc1eoN) [reply content/_guides/clients.md:43]: Wow good catch, it did that automatically 😓
Let me fix.

Comment (Nuc1eoN) [reply content/_guides/clients.md]: Fixed the localization for Revolution, as well as for LimeChat lmao

**Comment by kurahaupo** [comment]
I started #43 because I was frustrated by the name missing "IRC" from the end, and it sort of grew to include links to F-Droid and the source here on github as well.

Which would you like to use?

**Comment by swantzter** [comment]
duplicated by/improved by #43

**Pull request closed without merging**